.circleci/config.yml

```diff
@@ -30,28 +30,9 @@ executors:
       resource_class: windows.medium
     shell: bash.exe -eo pipefail
 
-parameters:
```

Comment:
We're keeping the circleci perf job running while the AWS instance is up, so we ditch the teardown job and the complicated configuration around that.

Reply:
This seems like a nice simplification. It'll take a much longer time for the job to finish now obviously but since they only run on master it's not like it is going to be blocking anything.